File: android/src/main/java/expo/modules/woosmapexpoplugingeofencingbatch/WoosmapExpoPluginGeofencingBatchModule.kt
package expo.modules.nativeconfiguration

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class WoosmapExpoPluginGeofencingBatchModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("WoosmapExpoPluginGeofencingBatch")

    Function("getApiKey") {
      return@Function "api-key"
    }
  }
}

File: ios/WoosmapExpoPluginGeofencingBatchModule.swift
import ExpoModulesCore

public class WoosmapExpoPluginGeofencingBatchModule: Module {
  public func definition() -> ModuleDefinition {
    Name("WoosmapExpoPluginGeofencingBatch")

    Function("getApiKey") { () -> String in
      "api-key"
    }
  }
}
